crates/cxx-qt-macro/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+# SPDX-FileContributor: Gerhard de Clercq <gerhard.declercq@kdab.com>
+# SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+[package]
+name = "cxx-qt-macro"
+version.workspace = true
+authors = ["Andrew Hayzen <andrew.hayzen@kdab.com>","Leon Matthes <leon.matthes@kdab.com>", "Gerhard de Clercq <gerhard.declercq@kdab.com>"]
+edition.workspace = true
+license.workspace = true
+description = "A macro for Qt/C++ interop in Rust"
+homepage = "https://kdab.github.io/cxx-qt/book/"
+readme = "README.md"
```

Comment:
there is no readme in this crate, so probably remove like other crates?

Reply:
Done